README.md

```diff
@@ -26,11 +25,11 @@ int main()
 
     TabSample tab;
     // explicit all_of(tab)
-    for(const auto& row : db.run(select(all_of(tab)).from(tab).where(true)))
+    for(const auto& row : db(select(all_of(tab)).from(tab).where(true)))
     {
         int64_t alpha = row.alpha;
-        bool beta = row.beta;
-        std::string gamma = row.gamma;
+        std::string beta = row.beta;
+        bool gamma = row.gamma;
```

Comment:
I am a bit confused as to why the test even works, but that might be due to the specific values in insert? Weird.

But yes, lines 20/21 should be changed as well, as should the definition in SampleTest.cpp